docs/source/about/installation.rst

```diff
@@ -210,24 +210,40 @@ Portfile
 
    .. code-block:: bash
 
-      sudo nano /opt/local/etc/macports/sources.conf
+      echo "file:///Users/$USER/ports" | sudo tee -a /opt/local/etc/macports/sources.conf
```

Comment:
Outside of a slightly ugly config file multiple entries of a source has no effect.

```bash
anon@Anons-MacBook-Air Sunshine % cat /opt/local/etc/macports/sources.conf
# MacPorts system-wide configuration file for ports tree sources.
#
# To change how MacPorts fetches base, see rsync_server and rsync_dir in
# macports.conf.

# To add a local source, add a "file://" entry.
#
#   Example: file:///Users/landonf/misc/MacPorts/ports
#
# To prevent a source from synchronizing when `port sync` is used,
# append "[nosync]" at the end.
#
#   Example: file:///Users/landonf/misc/MacPorts/ports [nosync]
#
# Note that MacPorts parses source URLs in order; when a port appears in
# multiple sources, it installs the first occurrence. For local sources
# to shadow remote ones, "file://" URLs must come before other URLs.

# A list of rsync mirrors is available at
# https://trac.macports.org/wiki/Mirrors#Portfiles.
#
# If an "rsync://" URL points to a .tar file, a signed .rmd160 file must
# exist in the same directory on the server and will be used to verify
# its integrity.
#
# For proper functionality of various resources (port groups, mirror
# sites, etc.), the primary MacPorts source must always be tagged
# "[default]", even if switched from the default "rsync://" URL.

rsync://rsync.macports.org/macports/release/tarballs/ports.tar [default]
file:///Users/anon/ports
file:///Users/anon/ports
file:///Users/anon/ports
anon@Anons-MacBook-Air Sunshine % sudo portindex
Creating port index in /Users/anon/Documents/Projects/Sunshine
Adding port packaging/macos

Total number of ports parsed:	1 
Ports successfully parsed:	1 
Ports failed:			0 
Up-to-date ports skipped:	0


```

Reply:
At a minimum I think a note should be added that this only needs to occur the first time someone is installing Sunshine.